src/index.js

```diff
@@ -1,7 +1,309 @@
+    height                : '100vh',
+    width                 : '100vw',
+    margin                : '0',
+  },
+  content : {
+    top                   : '50%',
+    left                  : '50%',
+    right                 : 'auto',
+    bottom                : 'auto',
+    marginRight           : '-50%',
+    transform             : 'translate(-50%, -50%)',
+    zIndex                : '3',
+    backgroundColor       : "gray",
+    border                : "2px solid rgb(120, 235, 140)"
+  },
+};
+
+const mHeader = {
+  marginBottom: "20px",
+}
+
+const mSubtitle = {
+  color: 'green',
+  paddingBottom: '8%',
+  textAlign: 'center'
+}
+
+
+// main app component
+class App extends React.Component {
+  
+  // contructor
+  constructor() {
+    super()
+    // state for app
+    this.state = {
+      markerPosition: { lat: 43.99528, lng: -72.69156 },
+      county: undefined,
+      town: undefined,
+      gameStarted: false,
+      giveUp: false,
+      borderLayer: L.geoJSON(BorderData),
+      countyLayer: L.geoJSON(CountyData),
+      modalOpen: false,
+      countyGuess: 'Addison',
+      correctGuess: false,
+      score: 100
+    };
+    // bind functions
+    this.moveMarker = this.moveMarker.bind(this);
+    this.clickStart = this.clickStart.bind(this);
+    this.startGame = this.startGame.bind(this);
+    this.handleGiveup = this.handleGiveup.bind(this);
+    this.openModal = this.openModal.bind(this);
+    this.afterOpenModal = this.afterOpenModal.bind(this);
+    this.closeModal = this.closeModal.bind(this);
+    this.handleGuess= this.handleGuess.bind(this);
+    this.getCounty = this.getCounty.bind(this);
+    this.getTown = this.getTown.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleGuess = this.handleGuess.bind(this);
+    console.log(this.state.borderLayer);
+  }
+
+  // function that moves the current marker to a new position
+  moveMarker() {
+    // use of object destructuring
+    {console.log('moved marker')}
+    const { lat, lng } = this.state.markerPosition;
+    this.setState({
+      markerPosition: {
+        lat: lat + 0.0001,
+        lng: lng + 0.0001, 
+      }
+    });
+  };
+
+  // fuction handles what happens when the start button is clicked
+  clickStart() {
+    console.log('clicked start');
+    // start the game
+    this.startGame();
+  }
+
+  // function that picks a random lat and long,
+  // centers the map on that spot, and zooms in
+  async startGame() {
+    const { borderLayer } = this.state;
+    console.log(borderLayer);
+    // there might be a better way to do this
+    const vtBounds = borderLayer.getBounds();
+    const xMax = vtBounds.getEast();
+    const xMin = vtBounds.getWest();
+    const yMax = vtBounds.getSouth();
+    const yMin = vtBounds.getNorth();
+
+    // find random lat and long 
+    // -> pull this out and generalize to find 
+    // random points in a given polygon/border?
+    let randomLatLon = () => {
```

Comment:
In terms of finding an easier solution, this could be done with a while loop instead of a recursive function. But your solution to finding a random lat/long coordinates is more legible due to your documentation. Although the comment on line 137 could be more descriptive in explaining its purpose in a recursive function.